editoast/src/schema/mod.rs

```diff
@@ -219,3 +219,31 @@ pub struct TrackEndpoint {
     #[derivative(Default(value = r#""InvalidRef".into()"#))]
     pub track: String,
 }
+
+#[derive(Debug, Derivative, Clone, Deserialize, Serialize)]
+#[derivative(Default)]
+pub enum Side {
+    #[serde(rename = "LEFT")]
+    Left,
+    #[serde(rename = "RIGHT")]
+    Right,
+    #[serde(rename = "CENTER")]
+    #[derivative(Default)]
+    Center,
+}
+
+#[derive(Debug, Derivative, Clone, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+#[derivative(Default)]
+pub struct Panel {
+    #[derivative(Default(value = r#""InvalidRef".into()"#))]
+    pub track: String,
+    #[derivative(Default(value = "0."))]
```

Comment:
What's the point of this line since the default value of a float is already `0.`?